gen/definitions/device_ha_pair.yaml

```diff
@@ -0,0 +1,156 @@
+---
+name: Device HA Pair
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/devicehapairs/ftddevicehapairs
+data_source_name_query: true
+doc_category: Device
+attributes:
+  - model_name: name
+    type: String
+    mandatory: true
+    description: The name of the access control policy.
```

Comment:
I believe this is the name of HA pair, not the policy

Reply:
changed, thanks